package.json

```diff
@@ -7,11 +7,12 @@
     "start-dev": "REACT_DEBUGGER='rndebugger-open --open --port 8081' npm start",
     "postinstall": "rndebugger-open",
     "test": "jest",
-    "test-redux": "jest src/store",
+    "test-redux": "jest src/store -- watch",
```

Comment:
Not familiar with jest, but just making sure - you're certain the flag is `dash dash space watch`? I'd have thought there would be no space, but asking just to be sure?

Reply:
I have no idea haha, this is how they have it in their documentaion

Reply:
👍